SemiLeptonicWVA/Utils/src/GenEventInfo.cc

```diff
@@ -264,6 +272,23 @@ GenEventInfo::produce(edm::Event& iEvent, const edm::EventSetup& iSetup)
 	  PUWeight = LumiWeights_.weight( Tnpv );
 	}
 
+
+
+	edm::Handle< LHEEventProduct > LHEEventInfo;
+        iEvent.getByLabel("source", LHEEventInfo);
+
+	std::vector<double> AQGCweights;
+        double originalWeight;
```

Comment:
change to:  double originalWeight = -1;
This will allow us to know exactly what value it starts at, and know if it ever gets reset in the code.